app/mailers/twilio_sender.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# TwilioSender: Methods to interact with Twilio REST API
+class TwilioSender
+  @client = Twilio::REST::Client.new(ENV['TWILLIO_API_ACCOUNT'], ENV['TWILLIO_API_KEY'])
+  def self.send_sms(patient, contents)
+    from = ENV['TWILLIO_MESSAGING_SERVICE_SID'] || ENV['TWILLIO_SENDING_NUMBER']
+    begin
+      @client.messages.create(
+        to: Phonelib.parse(patient.primary_telephone, 'US').full_e164,
+        body: contents,
+        from: from
+      )
+    rescue Twilio::REST::RestError => e
+      Rails.logger.warn e.error_message
+      patient.update(last_assessment_reminder_sent: DateTime.now)
```

Comment:
If this code is supposed to just be purely sending the messages and returned a true/false based on success, I don't think we want this update to be here.

Reply:
Good point -- these updates are redundant anyhow since the updates are already occurring after these sender methods are called.